### DISCUSSION TITLE
v1.115.0

### DISCUSSION THREAD

**Discussion by alextran1502** [immich-app/immich#12602] [Announcements]
# v1.115.0

> [!NOTE]
> We are working on redesigning the flows and layout of the mobile app to incorporate recently added features such as folders and tags, as well as making improvements to album management on the mobile app. We are gathering feedback in this [discussion](https://github.com/immich-app/immich/discussions/12597). We are excited to hear your feedback.

Welcome to release `v1.115.0` of Immich. This release focuses on cleaning up our backlog of issues. It is also worth noting that the [Cursed knowledge page](http://immich.app/cursed-knowledge/) has been updated with _even more cursed knowledge_. Please find some of the release highlights below:

- Administration button
- Manually link and unlink Live motion photos
- Default exclusion patterns
- Start-up folder checks
- Upload trash indicator

Notable bug fixes include:
- Signing out of the web now signs out of all tabs
- Long-standing issue regarding inaccurate album asset count has been fixed
- Search text is preserved when switching between "Context" and "File name or extension"
- Editing a person's name or birthdate on mobile has been fixed

### Administration button

The `Administration` button on the web is now moved inside the profile sheet to clean up the top app bar and improve the navigation experience for all screen sizes.

<img width="346" alt="image" src="https://github.com/user-attachments/assets/36389e3d-3fc9-4a4b-a480-f3165ec114bc">

### Manually link and unlink Live motion photos
Live photos (iOS) can now be manually linked an unlinked.

[Screencast from 09-11-2024 10:15:57 AM.webm](https://github.com/user-attachments/assets/07ec144a-af03-4827-8e6b-f2b4f30ec1f4)

### Default exclusion patterns
We have added default exclusion patterns whenever a new external library is created.  By default, we exclude the ```@eaDir``` folder Synology uses to store miscellaneous files and any files beginning with ```._```. Of course, you can edit these patterns if you want to, but this is highly unlikely.

![image](https://github.com/user-attachments/assets/1ec7fe82-c0f0-4049-a3fa-54e42ebd5d19)

### Startup folder checks

Immich will now _fail to start_ if it is unable to read/write to `upload/`, `library/`, `thumbs/`, `encoded-videos/` or `profile/`. Starting with this release, the server will write an `.immich` file to each of the previously listed folders. On subsequent restarts, it will _fail to start_ if it fails to locate the previously written `.immich` file. This will hopefully increase transparency around situations where the `UPLOAD_LOCATION` mount is incorrectly configured or unavailable.

### Upload trash indicator

When you upload a duplicate via the web and that duplicate is in the trash, you now will see an indicator for it, along with being able to open the asset in a new tab.

| Light | Dark |
| - | - |
| ![image](https://github.com/user-attachments/assets/78facde1-db58-49bf-833e-e53179082308) | ![image](https://github.com/user-attachments/assets/6a58bae3-0ccf-40dc-ad89-5b43da41faeb) |

- - - -

## Support Immich

<p align="center">
<img src="https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExbjY2eWc5Y2F0ZW56MmR4aWE0dDhzZXlidXRmYWZyajl1bWZidXZpcyZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/87CKDqErVfMqY/giphy.gif" width="450" title="SUPPORT THE PROJECT!"> 
</p>

If you find the project helpful, you can support Immich by purchasing a product key at [https://buy.immich.app](https://buy.immich.app).

Cheers! 🍻

### 🚀 Features
* feat(web): manually link live photos by @jrasm91 in https://github.com/immich-app/immich/pull/12514
* feat(web): unlink live photos by @jrasm91 in https://github.com/immich-app/immich/pull/12574
### 🌟 Enhancements
* feat(web): logout of all tabs by @jrasm91 in https://github.com/immich-app/immich/pull/12407
* feat(server): start up folder checks by @jrasm91 in https://github.com/immich-app/immich/pull/12401
* feat(web): responsive top navigation by @ben-basten in https://github.com/immich-app/immich/pull/12373
* feat(mobile): Add dismiss action on app_bar_dialog by @Lauritz-Tieste in https://github.com/immich-app/immich/pull/12511
* feat(web): select the EXIF timezone (if it exists) in dropdown by @C-Otto in https://github.com/immich-app/immich/pull/12495
* fix(web): show trash indicator by @jrasm91 in https://github.com/immich-app/immich/pull/12521
* feat(web): move search options into a modal by @ben-basten in https://github.com/immich-app/immich/pull/12438
* feat(server): default exclusion patterns by @etnoy in https://github.com/immich-app/immich/pull/12566
### 🐛 Bug fixes
* fix(server): remove hidden assets from albums by @jrasm91 in https://github.com/immich-app/immich/pull/12449
* fix(server): handle multiple hierarchical subjects by @jrasm91 in https://github.com/immich-app/immich/pull/12509
* fix(mobile): scroll thumb is hidden behind the tab navigation bar by @alextran1502 in https://github.com/immich-app/immich/pull/12512
* fix(web): select partner assets from timeline by @jrasm91 in https://github.com/immich-app/immich/pull/12517
* fix(web): preserve search text by @jrasm91 in https://github.com/immich-app/immich/pull/12531
* fix(web): show upload error message on network error by @jrasm91 in https://github.com/immich-app/immich/pull/12533
* fix(server): handle invalid directory item by @jrasm91 in https://github.com/immich-app/immich/pull/12534
* fix(web): leave assets selected after add to album by @jrasm91 in https://github.com/immich-app/immich/pull/12537
* fix(server): person repo methods by @jrasm91 in https://github.com/immich-app/immich/pull/12524
* fix(ml): batch axis not being added for recognition model by @mertalev in https://github.com/immich-app/immich/pull/12588
### 📚 Documentation
* docs: more cursed knowledge by @jrasm91 in https://github.com/immich-app/immich/pull/12529
* docs: proper value of word-based suggestions setting to setup VSCode by @sapachev in https://github.com/immich-app/immich/pull/12586
* docs: add server backup to First Steps by @mmomjian in https://github.com/immich-app/immich/pull/12555
### 🌐 Translations
* chore(web): update translations by @weblate in https://github.com/immich-app/immich/pull/12384

## New Contributors
* @pbustamantes made their first contribution in https://github.com/immich-app/immich/pull/12486
* @Lauritz-Tieste made their first contribution in https://github.com/immich-app/immich/pull/12511
* @jonathanjsimon made their first contribution in https://github.com/immich-app/immich/pull/12510
* @sapachev made their first contribution in https://github.com/immich-app/immich/pull/12586

**Full Changelog**: https://github.com/immich-app/immich/compare/v1.114.0...

<hr /><em>This discussion was created from the release <a href='https://github.com/immich-app/immich/releases/tag/v1.115.0'>v1.115.0</a>.</em>

**Comment by j-seixas** [comment]
Hey! Seems like an undocumented change was released -> https://github.com/immich-app/immich/pull/12587 reducing the minimum cuda version and therefore the nvidia driver version as well. This is good news for who is hosting in a linux machine where the stable branch of the driver is still on v535 like me, but it could have been in the release notes 😓 . Just randomly noticed this

**Reply by mertalev** [reply]
The minimum driver version was updated in the docs, but you're right that we could have mentioned it in the release notes too. Hope it makes things easier!
